docs/use/commands/pmm-admin.md

```diff
@@ -392,32 +392,36 @@ For example, if you want all collectors except `topmetrics`, specify:
 --enable-all-collectors --disable-collectors=topmetrics
 ```
 
-###### Limit `dbStats`, `collStats` and `indexStats`
+###### Limit `dbStats`, `collStats`, `indexStats` and `CurrentOp`
 
 By default, PMM decides the limit for the number of collections to monitor the `collStats` and `indexStats` collectors.
 
-You can also set an additional limit for the `collStats`, `indexStats`, `dbStats`, and `topmetrics` collectors with the `--max-collections-limit` parameter.
+You can also set an additional limit for the `collStats`, `indexStats`, `dbStats`, `topmetrics` and `CurrentOp` collectors with the `--max-collections-limit` parameter.
 
 Set the value of the parameter `--max-collections-limit` to:
 
 - 0: which indicates that `collStats` and `indexStats` can handle unlimited collections.
 - n, which indicates that `collStats` and `indexStats` can handle <=n collections. If the limit is crossed - exporter stops collecting monitoring data for the `collStats` and `indexStats` collectors.
 - -1 (default) doesn't need to be explicitly set. It indicates that PMM decides how many collections it would monitor, currently <=200 (subject to change).
 
+The `CurrentOp`  collector behaves differently:
+
+- It is not affected by the `--max-collections-limit` parameter.
+- Instead, it uses a separate `--max-current-ops-limit` parameter to control the number of current operations monitored.
```

Comment:
there are not such parameter and limit at all.
we have other limits: 
1. hardcoded limit to collect operations longer than a minute
2. operations in admin and local databases are ignored

I think it should be a separate